cnf/Makegap.in

```diff
@@ -48,632 +48,637 @@ endif
 
 # dependencies are automatically generated
 ariths.o: ../../Makefile @srcdir@/ariths.c @srcdir@/system.h config.h @srcdir@/globalstate.h \
- @srcdir@/code.h @srcdir@/scanner.h @srcdir@/gasman.h @srcdir@/hpc/atomic.h \
- @srcdir@/objects.h @srcdir@/gap.h @srcdir@/gvars.h @srcdir@/calls.h \
- @srcdir@/opers.h @srcdir@/ariths.h @srcdir@/bool.h @srcdir@/records.h \
- @srcdir@/precord.h @srcdir@/lists.h @srcdir@/stringobj.h @srcdir@/hpc/thread.h \
- @srcdir@/hpc/tls.h
+  @srcdir@/code.h @srcdir@/scanner.h @srcdir@/gasman.h @srcdir@/hpc/atomic.h \
+  @srcdir@/objects.h @srcdir@/gap.h @srcdir@/gvars.h @srcdir@/calls.h \
+  @srcdir@/opers.h @srcdir@/ariths.h @srcdir@/bool.h @srcdir@/records.h \
+  @srcdir@/precord.h @srcdir@/lists.h @srcdir@/stringobj.h @srcdir@/hpc/thread.h \
+  @srcdir@/hpc/tls.h
 	$(CC) $(CPPFLAGS) $(GMP_CFLAGS) $(CFLAGS) -o ariths.o -c @srcdir@/ariths.c
```

Comment:
There is a huge diff in `cnf/Makegap.in`. This is due to a difference between clange and gcc. I recommend recreating this file with GCC (real GCC, I mean, not the "gcc frontend for clang" that apple ships). On my Mac, I have the gcc 5 and 6 installed via fink, and updating `Makegap.in` like this does the trick:
```
cd cnf && touch Makefile && make CC=gcc-5
```